db/migrate/20151125155213_create_middleware_servers.rb

```diff
@@ -0,0 +1,17 @@
+class CreateMiddlewareServers < ActiveRecord::Migration
+  def change
+    create_table :middleware_servers do |t|
+      t.string :name       # server name generated from id
+      t.string :feed       # feed
+      t.string :ems_ref    # path
+      t.string :nativeid   # id
+      t.string :type_path
+      t.string :host
```

Comment:
Yeah, here I think you can just change this to `hostname` (no need for a separate migration).  If you need to fix your local DB, you can rollback and then come forward, or if you don't want to lost the data, just do a simple `ALTER TABLE`